usaspending_api/api_contracts/contracts/v2/agency/toptier_code/program_activity.md

```diff
@@ -0,0 +1,94 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# List Program Activity [/api/v2/agency/{toptier_code}/program_activity/{?fiscal_year,filter,order,sort,page,limit}]
+
+Returns a list of Program Activity in the Agency's appropriations for a single fiscal year
+
+## GET
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "number"
+            }
+    + Parameters
+        + `toptier_code`: 086 (required, number)
+            The toptier code of an agency (could be a CGAG or FREC) so only numeric character strings of length 3-4 are accepted.
+        + `fiscal_year` (optional, number)
+            The desired appropriations fiscal year. Defaults to the current FY.
+        + `filter` (optional, string)
+            This will filter the Program Activity by their name to those matching the text.
+        + `order` (optional, enum[string])
+            Indicates what direction results should be sorted by. Valid options include asc for ascending order or desc for descending order.
+            + Default: `desc`
+            + Members
+                + `desc`
+                + `asc`
+        + `sort` (optional, enum[string])
+            Optional parameter indicating what value results should be sorted by.
+            + Default: `obligated_amount`
+            + Members
+                + `name`
+                + `obligated_amount`
+                + `gross_outlay_amount`
+        + `page` (optional, number)
+            The page number that is currently returned.
+        + `limit` (optional, number)
+            How many results are returned.
+            + Default: 10
+
++ Response 200 (application/json)
+    + Attributes
+        + `toptier_code` (required, string)
+        + `fiscal_year` (required, number)
+        + `limit` (required, number)
+        + `page_metadata` (required, PageMetadata, fixed-type)
```

Comment:
Since this table will use a pager element (rather than infinite scroll or next / prev buttons) we'll need a `count` of total results. We don't really need `next`, `previous`, `hasNext`, or `hasPrevious` for USAS frontend purposes, but no harm in including them.

Reply:
Since we have the count endpoints to go along with these list endpoints should the total still be added into the page_metadata here?

Reply:
True, we could pass down the totals from the tab component!